src/phasorpy/phasor.py

```diff
@@ -3070,11 +3099,152 @@ def _median(
     return numpy.nanmedian(real, **kwargs), numpy.nanmedian(imag, **kwargs)
 
 
-def _median_filter(
+def _median_filter_2d(
+    real: ArrayLike,
+    imag: ArrayLike,
+    /,
+    *,
+    repeat: int = 1,
+    size: int = 3,
+    axes: Sequence[int],
+) -> tuple[NDArray[Any], NDArray[Any]]:
+    """Return the phasor coordinates after applying a 2D median filter.
+
+    Parameters
+    ----------
+    real : ndarray
+        Real components of the phasor coordinates.
+    imag : ndarray
+        Imaginary components of the phasor coordinates.
+    axes : int or sequence of int
```

Comment:
Description does not match typing.

Reply:
Thanks Christoph, I made some last minute changes to avoid checking some variables and forgot to change the description.